### ISSUE TITLE
Fix header CTA text to meet WCAG AA for contrast ratio

### ISSUE THREAD

**Issue by Kernix13** [Kernix13/CY-portfolio-template#7]
The button in the header labeled "Get in touch" is below 3 for the contrast ratio. The background color of the button is `#ff3258`, the text color is `#e5e5e6da`. If the text color was changed to `#FFFFFF` it would still fail. A color of `#202020` or darker would make it pass so it should be changed. Or the background color needs to be darker.
